server/server.go

```diff
@@ -693,9 +695,18 @@ func (s *Server) GetConfig() *config.Config {
 	if err != nil {
 		return cfg
 	}
-	payload := make(map[string]string)
+	payload := make(map[string]interface{})
 	for i, sche := range sches {
-		payload[sche] = configs[i]
+		var config map[string]interface{}
+		err := schedule.DecodeConfig([]byte(configs[i]), &config)
+		if err != nil {
+			log.Error("failed to decode scheduler config",
+				zap.Reflect("config", configs[i]),
+				zap.Reflect("scheduler", sche),
```

Comment:
Can we use String here?

Reply:
Done

Reply:
Actually, I mean something like
```suggestion
				zap.String("scheduler", sche),
```

Reply:
I have changed the field type, please take
 a look.